app/experimenter/nimbus-ui/src/index.tsx

```diff
@@ -24,7 +24,7 @@ try {
     <React.StrictMode>
       <AppErrorBoundary>
         <ApolloProvider client={createApolloClient()}>
-          <App basepath="/experiments/nimbus" />
+          <App />
```

Comment:
Vague nit possibly not worth thinking more than 2 seconds about: I was thinking this would eventually be supplied in the meta-header config from Django. That way, the base path would be kept in sync with URL routes on the server. But I guess we shouldn't be changing it all that often, so it could stay a constant.